addons/html_builder/static/src/website_builder/plugins/website_edit_service.js

```diff
@@ -141,6 +178,18 @@ registry.category("services").add("website_edit", {
                         // reinserted upon undo of the option's action.
                         el.dataset.skipHistoryHack = "true";
                     },
+                }),
+                patch(publicInteractions.constructor.prototype, {
+                    shouldStop(...args) {
```

Comment:
honestly, the `...args` is not really necessary and makes it harder to understand. 
```js
                    shouldStop(el, interaction) {
                        if (super.shouldStop(el, interaction)) {
                            if (this.isRefreshing) {
                                return interaction.interaction.shouldStop();
                            }
                            return true;
                        }
                        return false;
                    },
```